qemu/tests/cfg/slof_boot.cfg

```diff
@@ -0,0 +1,12 @@
+- slof_boot: install setup image_copy unattended_install.cdrom
+    type = boot
+    restart_vm = yes
+    kill_vm_on_error = yes
+    login_timeout = 240
```

Comment:
Do we need to increase login_timeout if both pysical_block_size and logical_block_size are 512?